examples/benchmarks/resnet_imagenet/tests/test_data.py

```diff
@@ -1,14 +1,19 @@
 # Copyright 2022 MosaicML Examples authors
 # SPDX-License-Identifier: Apache-2.0
 
+import os
 import sys
 
 import pytest
 import torch
 
-from examples.resnet_imagenet.data import (build_imagenet_dataspec,
-                                           check_dataloader)
-from examples.resnet_imagenet.tests.utils import SynthClassificationDirectory
+# Add tests folder root to path to allow us to use relative imports regardless of what directory the script is run from
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+# Add folder root to path to allow us to use relative imports regardless of what directory the script is run from
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Why twice?

Reply:
they're different! one is the tests folder, and one is the source folder